Program.cs

```diff
@@ -55,6 +59,7 @@ public static void AddPerson()
             people.Add(newPerson);
         }
 
+        // changing information about a person on the book
         public static void UpdatePerson()
         {
             PersonList();
```

Comment:
Its name should be ListPeople(). Because methods should start with verbs!

Reply:
done 👍